style.css

```diff
@@ -65,8 +61,7 @@ body {
     overflow: hidden;
     position: relative;
     margin : 0 auto;
-    top : 20px;
-    //margin-top : -50px;
+    top : 10px;
```

Comment:
I think those are not the major parts to be modified. but any way I'll do it..is there any thing you want me to modify.